packages/api-form-builder/__tests__/formSubmissionSecurity.test.ts

```diff
@@ -170,6 +187,12 @@ describe("Forms Submission Security Test", () => {
                             ...formA,
                             savedOn: expect.stringMatching(/^20/),
                             fields: expect.any(Array),
+                            steps: [
+                                {
+                                    title: null,
```

Comment:
Confused a bit why we sometimes have `""`, and sometimes `null` here. Am I missing something here?

But in any case, if we add the "required" validator I mentioned in my prev comments, this should no longer be an issue.